driverinterface.sh

```diff
@@ -1,13 +1,8 @@
 #!/bin/sh 
-cd /home/ucrsolarcar/UCRSolarCar2023/Driver-Interface-2022
 
-res=$(git pull origin main | grep Already)
```

Comment:
You have to add the formatting before the code compiles. The current code is responsible for compiling the code on the raspberry pi and running the application automatically. You will have to add your steps into the current script.

Reply:
Apologies for this. I wasn't sure what that code was doing. I have restored the original code and added the clang-format script to it.